requirements.txt

```diff
@@ -83,7 +75,8 @@ scikit-image==0.14.2
 # OpenAI gym is only needed in chapter 18.
 # There are a few dependencies you need to install first, check out:
 # https://github.com/openai/gym#installing-everything
-gym[atari]==0.10.9
+# gym[atari]==0.10.9
+gym[atari,box2d,classic_control]==0.10.9
```

Comment:
`RUN pip install "gym[atari,box2d,classic_control]"` as it was in the original Dockerfile created a peer dep conflict 
for cloudpickle - `tfp-nightly 0.9.0.dev20190926 has requirement cloudpickle==1.1.1` and `gym has requirement cloudpickle==1.1.2` so i froze the later. (it was un-frozen in Dockerfile originally, but always warned on/after ~Sept 21, 2019, when i started this book, fyi)

Feedback welcome.